pycloudlib/util.py

```diff
@@ -287,3 +288,18 @@ def _safe_int(possible_int):
         return int(possible_int)
     except (ValueError, TypeError):
         return None
+
+
+def create_tag(tag):
```

Comment:
good idea, can we also provide a strict=False param or time_suffix=True or something else to allow avoiding appending the datetime suffix